components/si-labs/CP2105.stanza

```diff
@@ -20,32 +20,6 @@ public pcb-component component :
   description  = "IC SGL USB-DL UART BRIDGE 24QFN"
   mpn          = "CP2105-F01-GM"
 
-  pin REGIN
-  pin VDD
-  pin RI_SCI
-  pin GND
-  pin VBUS
-  pin D+
-  pin D-
-  pin VIO
-  pin nRST
-  pin CTS_ECI
-  pin RTS_ECI
-  pin RXD_ECI
-  pin TXD_ECI
-  pin DSR_ECI
-  pin DTR_ECI
-  pin DCD_ECI
-  pin RI_ECI 
-  pin CTS_SCI
-  pin RTS_SCI
-  pin RXD_SCI
-  pin TXD_SCI
-  pin DSR_SCI
-  pin DTR_SCI
-  pin DCD_SCI
-  pin EPAD
-
   pin-properties :
```

Comment:
@ariel-hi Just added a Linear task to take all pin:Ref's in a pin-properties: block to be automatically promoted to the component level as well (making the explicit definition redundant). This is ok until that PR is done.